.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Python tests
+
+on:
+  push:
+    branches: 'master'
+  pull_request:
+    branches: 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
This shouldn't be conditional. This file should always be present. If it's not, that's a bug. If it's deleted from the repo accidentally, this should fail and not be skipped silently.

Reply:
ok, i remove 'if'